WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingPaymentMethod/WooShippingPaymentMethodsView.swift

```diff
@@ -18,13 +19,41 @@ struct WooShippingPaymentMethodsView: View {
                 paymentMethodList
             }
 
+            HStack(alignment: .top) {
+                Image(systemName: "info.circle")
+                Text(String(format: Localization.note, viewModel.storeOwnerUsername))
+                    .frame(maxWidth: .infinity, alignment: .leading)
+            }
+            .font(.footnote)
+            .foregroundStyle(Color.primary)
```

Comment:
This view is inconsistent in the design. I'm using the style used in both the design for existing payment methods and the one for shop managers.